evergreen.yml

```diff
@@ -283,7 +264,7 @@ functions:
     params:
       directory: src/genny
       revisions:
-        mongo: cf8ddbdf99ae7fb2d96ca9f350224ffa193e40b3
+        mongo: 6e2dcc5a39eb2de9d2e8209271115c078ae16470
```

Comment:
Same here about specifying in evergreen, ideally the genny python can handle all of this now?

Reply:
This is required and is not part of resmoke. This is required for the run-genny used in evergreen.yml.